### DISCUSSION TITLE
3/25 Discussion with Prof

### DISCUSSION THREAD

**Discussion by willschenk** [ellenguyen/CIS4496_EY#11] [General]
DISCUSSION Mon (03/25)
Asked model architecture approach:
train a model to auto labelling other areas, then retrain to detect damages
OR detect buildings then classify 4 classes -> easier
Prof: has to be building - type - damage in order

Asked model training ordering:
type & damage are correlated -> should go with 2nd model approach
should not use the same training data to avoid overfitting

Jovan: 
split up to get 3 different types of labels each, might have some mistakes when labeling -> then try predicting annotations
could detect all 3 at once = multi-class labeling -> array of 3 prediction probabilities [building, type, damage]
